ProjFS.Mac/Scripts/Build.sh

```diff
@@ -5,6 +5,11 @@ if [ -z $CONFIGURATION ]; then
   CONFIGURATION=Debug
 fi
 
+VERSION=$2
+if [ -z $VERSION ]; then
+  VERSION="0.2.173.2"
```

Comment:
Do we expect to update this manually on occasion?

Reply:
"0.2.173.2" is a special version number that was picked years ago that means that VFS4G was built on a developers machine.  There are no plans to ever update it.